.github/actions/setup-linux/action.yml

```diff
@@ -30,16 +30,20 @@ runs:
         fi
 
     - name: Log in to ECR
-      shell: bash
+      uses: nick-fields/retry@3e91a01664abd3c5cd539100d10d33b9c5b68482
       env:
         AWS_RETRY_MODE: standard
         AWS_MAX_ATTEMPTS: "5"
         AWS_DEFAULT_REGION: us-east-1
-      run: |
-        AWS_ACCOUNT_ID=$(aws sts get-caller-identity|grep Account|cut -f4 -d\")
-        retry () { "$@"  || (sleep 1 && "$@") || (sleep 2 && "$@") }
-        retry aws ecr get-login-password --region "$AWS_DEFAULT_REGION" | docker login --username AWS \
-            --password-stdin "$AWS_ACCOUNT_ID.dkr.ecr.$AWS_DEFAULT_REGION.amazonaws.com"
+      with:
+        shell: bash
+        timeout_minutes: 5
+        max_attempts: 3
+        retry_wait_seconds: 30
+        command: |
```

Comment:
Lint will bark on this as `command` has the wrong indentation I think :D

Reply:
I believe this is the correct indentation, but we'll see :)

Reply:
Ah, you're right. This is the correct indentation.